packages/create-cloudflare/e2e-tests/pages.test.ts

```diff
@@ -1,63 +1,67 @@
 import { existsSync, mkdtempSync, realpathSync, rmSync } from "fs";
+import crypto from "node:crypto";
 import { tmpdir } from "os";
 import { join } from "path";
 import { FrameworkMap } from "frameworks/index";
 import { readJSON } from "helpers/files";
+import { fetch } from "undici";
 import { describe, expect, test, afterEach, beforeEach } from "vitest";
 import { keys, runC3 } from "./helpers";
 import type { RunnerConfig } from "./helpers";
+import spawn from "cross-spawn";
 
 /*
 Areas for future improvement:
-- Make these actually e2e by verifying that deployment works
 - Add support for frameworks with global installs (like docusaurus, gatsby, etc)
 */
 
+type FrameworkTestConfig = RunnerConfig & {
+	expectResponseToContain: string;
```

Comment:
I guess `expectResponseFromDeploymentToContain` would be a bit too long? 😅

Reply:
omg you read my mind! I typed that out originally as I struggled to name this but yeah, it feels just a little too long :D